File: app/src/main/java/com/neome/java/api/meta/base/dto/DefnFieldTime.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnFieldTime extends DefnFieldEditable
{
  @Nullable
  public MetaIdField defaultFieldId;

  @Nullable
  public DefnBuildTime defaultValue;

  @Nullable
  public DefnBuildTime defaultVar;

  @Nullable
  public String displayDateFormat;

  @Nullable
  public DefnBuildTime max;

  @Nullable
  public MetaIdField maxFieldId;

  @Nullable
  public DefnBuildTime maxVar;

  @Nullable
  public DefnBuildTime min;

  @Nullable
  public MetaIdField minFieldId;

  @Nullable
  public DefnBuildTime minVar;

  @Nullable
  public Boolean showSecond;

  @Nullable
  public MetaIdField showSecondFieldId;

  @Nullable
  public Boolean showSecondVar;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldTime.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnBuildTime
import com.neome.api.meta.base.dto.DefnFieldEditable
import com.neome.api.meta.base.Types.MetaIdField

interface DefnFieldTime : DefnFieldEditable
{
  val defaultFieldId: MetaIdField?
  val defaultValue: DefnBuildTime?
  val defaultVar: DefnBuildTime?
  val displayDateFormat: String?
  val max: DefnBuildTime?
  val maxFieldId: MetaIdField?
  val maxVar: DefnBuildTime?
  val min: DefnBuildTime?
  val minFieldId: MetaIdField?
  val minVar: DefnBuildTime?
  val showSecond: Boolean?
  val showSecondFieldId: MetaIdField?
  val showSecondVar: Boolean?
}